recipes/bzip2/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: bzip2
+  version: 1.0.6
+
+source:
+  fn: bzip2-1.0.6.tar.gz
+  url: http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz
+  md5: 00b516f4704d4a7cb50a1d97e6e8e15b
+
+build:
+  number: 4
+
+test:
+  commands:
+    - bzip2 --help 2> /dev/null
+
+
+about:
+  home: http://www.bzip.org/
+  license: BSD
```

Comment:
Please add `license_family: BSD`.

Also please change `license` to `BSD-like` or `bzip2`, whichever is the preference of others.